core/src/trezor/messages.py

```
@@ -414,6 +414,138 @@
+            self,
+            *,
+            raw_tx: "bytes",
+            address_n: "list[int] | None" = None,
+            data_initial_chunk: "bytes | None" = None,
+            coin_type: "bytes | None" = None,
+            data_length: "int | None" = None,
+        ) -> None:
+            pass
+
+        @classmethod
+        def is_type_of(cls, msg: Any) -> TypeGuard["BenfenSignTx"]:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)